docs/core-concepts/packages.md

```diff
@@ -0,0 +1,143 @@
+    "netstandard1.5": {}
+  }
+}
+```
+
+In the following example, the `Microsoft.NETCore.App` metapackage is referenced, which is used for creating apps and libraries that are intended to run on and take full advantage of .NET Core. It provides access to a larger set of libraries than are provided by `NETStandard.Library`.
+
+```json
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0"
+  },
+  "frameworks": {
+    "netcoreapp": {}
+  }
+}
+```
+
+Target Frameworks
+=================
+
+.NET Core packages each support a set of target frameworks, declared with target framework folders (within the lib and ref folders mentioned earlier). Target frameworks describe an available API set (and potentially other characteristics) that you can rely on when you target a given framework. They are versioned as new APIs are added.
+
+For example, [System.IO.FileSystem](https://www.nuget.org/packages/System.IO.FileSystem) supports the following target frameworks:
+
+- net46
+- netstandard1.3
+- 6 different Xamarin platforms (for example, xamarinios10)
+
+It is useful to contrast the first two of these target frameworks, since they are examples of the two different ways that target frameworks are defined.
+
+The `net46` framework represents the available APIs in the .NET Framework 4.6. You can produce libraries that were compiled with the .NET Framework 4.6 reference assemblies and then distribute those libraries in NuGet packages in a net46 lib folder. It will be used for apps that target the .NET Framework 4.6 or are compatible with it. This is how all framework targets have traditionally worked.
+
+The `netstandard1.3` framework is a package-based target framework, so doesn't expose or describe any APIs in the absense of packages that target that framework. It is the interplay of the framework and the packages that define the set of APIs that are available when targeting the framework.
+
+Package-based Target Frameworks
+===============================
+
+There is a two-way relationship between frameworks and packages. The first part is defining the APIs available for a given framework, for example 'netstandard1.3'. Continuing with that example, there are no APIs that are inherently part of `netstandard1.3`. All of the `netstandard1.3` APIs come from packages that target `netstandard1.3` or compatible frameworks, such as `netstandard1.0`. These packages and the APIs they expose define the framework. Again, without the presence of packages that target `netstandard1.3`, there would be no APIs available for the framework.
+
+The second part of the relationship is asset selection. Packages can contain assets for multiple frameworks. Given a reference to a set of packages and/or metapackages, the target framework is needed to determine which asset that should be selected, for example `net46` or `netstandard1.3`. It important to select the correct asset. For example, a `net46` asset is not likely to be compatible with .NET Framework 4.0 or .NET Core 1.0.
```

Comment:
which asset that should be selected --> which asset should be selected
It important to select  ---> It is important to select